README.md

```diff
@@ -83,7 +62,53 @@ $ rake ids:update
 This rake task queries the NRDB API and creates/updates identities as appropriate.
 Identities not in the database are stripped out of ABR uploads to avoid errors.
 
-## Deploy as a web server
+## Feature flags
+
+[Flipper](https://github.com/jnunemaker/flipper) is included to give the option to hide or disable features which are
+incomplete. This lets you make changes in smaller increments, while still being able to deploy to a production
+environment with your feature hidden.
+
+You can enable or disable a feature for all development environments by editing
+[development.rb](config/environments/development.rb).
+
+Code like this will update the database to enable the feature when the app starts up, but not in situations where the
+database might not be fully available (eg. when initialising the database for the first time in a rake task):
+
+```ruby
+Rails.application.configure do
+
+   # This block should already exist and contain other configuration here...
+
+   if defined?(Rails::Server)
```

Comment:
i saw that locally but was unable to actually see the feature enabled.  re-built, down/up, same deal.

Reply:
When you create a tournament, you need to tick self registration and the new NRDB-based registration as well. I'm not sure if there's a better way to set up those options. Can you see both options when you create a tournament? Are you definitely running with the development environment?

Reply:
As I shared on Discord, the suggested change to config/environments/development.rb is not doing the trick.  I had to manually enable the feature in the rails console (which at least let me test this out!).

Reply:
I've changed how it enables the feature. I reproduced the problem you saw by running the app in Docker, and it should work now. I've tested it from a clean Docker with `docker compose down -v`.